static/app/views/externalRedirect/index.tsx

```diff
@@ -0,0 +1,92 @@
+import {useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+import dompurify from 'dompurify';
+
+import Panel from 'sentry/components/panels/panel';
+import {t, tct} from 'sentry/locale';
+import {isUrl} from 'sentry/utils';
+
+function ExternalRedirect() {
+  const [count, setCount] = useState<number>(5);
+  const [redirectUrl, setRedirectUrl] = useState<string | null>(null);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    const queryParams = new URLSearchParams(window.location.search);
```

Comment:
You can get the query string value by doing this instead (outside of the useEffect):

```js
const location = useLocation()
const url = location.query.url
```